core-bundle/contao/classes/DropZone.php

```diff
@@ -29,8 +29,11 @@ public function generateMarkup()
 		$strAccepted = implode(',', array_map(static function ($a) { return '.' . $a; }, StringUtil::trimsplit(',', strtolower(Config::get('uploadTypes')))));
 
 		// Add the scripts
-		$GLOBALS['TL_CSS'][] = 'assets/dropzone/css/dropzone.min.css';
-		$GLOBALS['TL_JAVASCRIPT'][] = 'assets/dropzone/js/dropzone.min.js';
+		if (!System::getContainer()->get('contao.routing.scope_matcher')->isBackendRequest())
+		{
+			$GLOBALS['TL_CSS'][] = 'assets/dropzone/css/dropzone.min.css';
+			$GLOBALS['TL_JAVASCRIPT'][] = 'assets/dropzone/js/dropzone.min.js';
+		}
```

Comment:
Why is that ever needed, if the scripts/styles are always added in the template?

Reply:
iirc some users use this widget in the front end too.

Reply:
The class is meant to use in the back end only:

https://github.com/contao/contao/blob/19ddbb7c9cbd181b9c64b10ee982c07ae4cb5a36/core-bundle/contao/classes/DropZone.php#L13-L15

Reply:
Yes, but as I said, iirc some are using it in the front end - which would then break when updating to the next bugfix version. But if we are ok with that I will remove the check.

Reply:
`class DropZone extends FileUpload` and `class FileUpload extends Backend`. I doubt that anyone uses that in the front end.

Reply:
Removed in https://github.com/contao/contao/pull/8460/commits/5aed21dc87067a23116b36a513ad384b930724b2